getting-started/web-app.md

```diff
@@ -7,6 +7,7 @@ slug: getting-started/web-app
 tags: get,started,first,steps,web,app,template
 published: true
 position: 5
+previous_url: /getting-started/server-blazor
```

Comment:
It might be a good idea to add a temporary note to explain the redirect in the text. If people are trying to open the old URL, they might be confused why they're landing here.

Reply:
It's highly unlikely that someone will be starting a new Blazor app on .NET 6.

Reply:
Marking this as resolved as there is no need for such a note.